styleguide/source/_patterns/03-organisms/article-header.twig

```diff
@@ -1,7 +1,7 @@
 {%  set link, heading, social = articleHeader.link, articleHeader.heading, articleHeader.social %}
 
 <header class="ama__article-header">
-    {% include "@atoms/link.twig" %}
+    <h4>{% include "@atoms/link.twig" %}</h4>
```

Comment:
was the h4 added or deleted? because when I pull your PR it's not there

Reply:
I'm not sure I understand.

Reply:
H4 was added for it's style. I firstly tried to @extend ama__h4, but that lead to a block <a>, so I used the wrapping h4.. Hopefully that's what was needed.

Reply:
We shouldn't be using the `<h4>` tag since that would lead to heading being used out of order and inappropriately which is bad for accessibility and SEO. We should be `@extend ama__h4` and modifying the rest of the `&__link` styles for this to make it look the way the UX wants.

Reply:
The reason I used <h4> wrapper was because the `<a>` was stretching across (as a block element) as in if I put my cursor over the right side of the area is would turn into a hand. I put a pink border to show this action. Would you rather this?

![screen shot 2018-02-09 at 5 46 13 pm](https://user-images.githubusercontent.com/231467/36053864-7f679c78-0dc1-11e8-8e4c-c486e085efbc.png)

Reply:
I'd rather something like:
```
@extend %ama__h4;
display: inline-block;
```

Reply:
I tried that -- It doesn't work with child "flex items"

Reply:
On a side note, the order of header tags doesn't affect SEO or Accessibility. When using flex, it's typical to use wrappers.